apps/pie-microsite/src/_includes/page-tabs.njk

```diff
@@ -0,0 +1,15 @@
+<nav aria-label="Secondary navigational tabs" class="c-tabs">
+    <ul>
+        {%- for entry in collections.all | eleventyNavigation(navKey) %}
```

Comment:
How does the `eleventyNavigation` filter handle parameters? I had a quick look at the docs but couldn't really see anything

Reply:
what kind of parameters were you thinking of?

Reply:
`navKey` looks like one to me in this case, although I might just be misreading it

Reply:
ah I see - sorry, so any values declared in a file's front matter (the bit at the top of the markdown file) is available in any layout/template used on that page. 

I believe passing that key to eleventyNavigation is helping generate a relative url of /colour or /spacing for the two pages that currently have tabs (Kevin may know a little more)

Reply:
I will remind myself of the API for that plugin tomorrow

Reply:
yeah so this assigns a parent key to the tab links i.e. color/{tabName}

https://www.11ty.dev/docs/plugins/navigation/#example-get-just-one-branch